repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/openstack/OpenStackAuthenticationHeaderManager.java

```diff
@@ -189,6 +189,7 @@ private void setTenant() {
             }
         } else if (!sendAllTenantIds && sendTenantIdQuality) {
             if (!this.isDelagable && !this.isTenanted) {
+                //TODO: Should this, or any other, instance of putHeader be changed to appendHeader?
```

Comment:
There should probably be a new story to review all uses of `putHeader()` and `appendHeader()` just to make sure the right one is being used.